src/contractStandards/ERC20WithDataABI.ts

```diff
@@ -0,0 +1,352 @@
+export const ERC20WithDataABI = [
```

Comment:
Did you convert JSON to TypeScript? Might consider just storing it as `.json` and importing the JSON file as an object, if that requires less steps. Up to you since I don't know the exact workflow to go from Solidity to here.

Reply:
Good idea, will just do `import ERC20WithDataABI from 'path/to/solidity/build/contracts/ERC20WithData.json';`